src/utils/attendee.js

```diff
@@ -27,6 +27,8 @@
  * @return {string} URI without a mailto prefix
  */
 export function removeMailtoPrefix(uri) {
+	if (typeof uri !== 'string') return ''
```

Comment:
Please wrap your ifs
```suggestion
	if (typeof uri !== 'string')  {
	     return ''
	}
	     
```

Reply:
Thanks! Will do next time I submit!